Solutions/Marain.ContentManagement.Tenancy.AzureCosmos/Marain.ContentManagement.Tenancy.AzureCosmos.csproj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="Current">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <PackageLicenseExpression>AGPL-3.0-or-later</PackageLicenseExpression>
```

Comment:
Is it wise to use the `or-later` bit? I've always been a bit mystified when projects declare that any future version of some license can be used in place of the current one because you have no idea what you're committing yourself to.

Reply:
Yes; this is a difficult area. There is much debate on the matter. You can always change the license "from this point forward" if you don't like it, of course.

In the general case, this gives us more flexibility and compatibility.

Reply:
Is the concern that without this, an component that hasn't been updated for a while because it's fine as it is might become unusable to someone because it's tied to the wrong (old) version of a license?

Reply:
That is correct.